README.md

```diff
@@ -58,15 +43,13 @@ Clone the FCCSW repository: **replace \<username\> by your github username**
 
```

Comment:
here we keep "export FCCSW=$FCCBASE/FCCSW" while "export FCCBASE=$PWD" has been previously removed
I think the environment variable FCCSW is not needed anymore